docs/types-and-platforms/server-types/paper.md

```diff
@@ -85,6 +85,24 @@ Extra variables:
 - `USE_FLARE_FLAGS=false` : set to true to add appropriate flags for the built-in [Flare](https://blog.airplane.gg/flare) profiler
 - `PURPUR_DOWNLOAD_URL=<url>` : set URL to download Purpur from custom URL.
 
+### Leaf
+
+A [Leaf server](https://www.leafmc.one/) is a Paper fork focused on performance improvements and low-level optimizations for smoother gameplay.
+
+To use a Leaf server, set the environment variable `TYPE` to `"LEAF"`.
+
+    -e TYPE=LEAF
+
+!!! note
+
+    The `VERSION` variable is used to select the Minecraft version to run.  
+    To specify a particular Leaf build, use `LEAF_BUILD`.  
+    You can also override the download source with `LEAF_DOWNLOAD_URL`.

```

Comment:
Support for LEAF_DOWNLOAD_URL isn't included. Just remove this sentence for now.